### ISSUE TITLE
Rsync not working

### ISSUE THREAD

**Issue by tylernchls** [lholmquist/angular-web-app#3]
CMD:  npx nodeshift watch --strictSSL=false

Error Message: Connection refused.  cannot use rsync-daemon: rsync not available in container

Any thoughts?

**Comment by lholmquist** [comment]
what is the environment you are running in?

**Comment by tylernchls** [comment]
Windows 10
cwrync installed

**Comment by lholmquist** [comment]
We are actually thinking of removing the `watch` command from nodeshift. https://github.com/nodeshift/nodeshift/issues/280

could you tell me if you still get an error using straight up `oc rsync`, which is what nodeshift is using under the covers anyway

something like `oc rsync --progress --watch --no-perms=true ./src POD_NAME:/opt/app-root/src/src`

**Comment by tylernchls** [comment]
Ya still get same error 
![capture](https://user-images.githubusercontent.com/22807079/53770889-8f7e9e80-3e95-11e9-98e4-719957567b39.PNG)

**Comment by lholmquist** [comment]
I don't think it is a problem with the application, it seems it could be a problem with how the oc client does rysnc'ing on windows.  you might want to do a search for "oc rsync + windows"

**Comment by tylernchls** [comment]
Ya not to sure only thing I could find was that cwRsync must be installed on windows. Ill keep searching, thanks for the help. Will update if i figure out solution for window users
